pkg/domain/infra/tunnel/containers.go

```diff
@@ -819,6 +820,14 @@ func (ic *ContainerEngine) ContainerRun(ctx context.Context, opts entities.Conta
 	}
 
 	// Attach
+	if opts.SigProxy {
+		terminal.ProxySignalsRemote(con.ID, func() error {
+			killOpts := entities.KillOptions{All: false, Latest: false, Signal: "TERM"}
```

Comment:
This isn't correct, we don't always proxy TERM. You need to proxy the signal that was received.

Reply:
Yeah, I eventually had to pass the `syscall.Signal` and convert it to string. I am pretty sure there is a better way to convert it to string so let me know what I'm missing.

Thanks.